agent/config/pbench-agent-default.cfg

```diff
@@ -21,7 +21,7 @@ logger_type = file
 
 [results]
 user = pbench
-host_path = http://%(pbench_result_redirector)s/pbench-archive-host
+host_path = http://%(pbench_results_redirector)s/pbench-archive-host
```

Comment:
Where is `host_path` used?

Reply:
That, actually, seems to be a really good question. I find no trace of the string host_path anywhere in the source tree. And, furthermore, since `pbench_result_redirector` ought to be defined to the server hostname, the path `http://{host}/pbench-archive-host` (e.g., `/var/www/html/pbench-archive-host`) doesn't actually exist on a Pbench server.

This line was last changed about 3 years ago by Pete, but the PR says nothing about it so I assume (I didn't look) that it was a minor refactoring or rearrangement.

Reply:
And in fact, since the spelling error goes back at least to 2019 and nobody's complained, we might even consider that proof that nobody's used it?

Reply:
Okay, went "whole-hog."

Reply:
@webbnh, do we need more discussion on this change?